_episodes/02-setup.md

```diff
@@ -41,11 +41,13 @@ in a later lesson will include this information.
 >
 > As with other keys, when you hit <kbd>Return</kbd> on your keyboard,
 > your computer encodes this input as a character.
-> For reasons that are long to explain, different operating systems
-> use different character(s) to represent the end of a line.
+> Different operating systems use different character(s) to represent the end of a line.
 > (You may also hear these referred to as newlines or line breaks.)
 > Because Git uses these characters to compare files,
-> it may cause unexpected issues when editing a file on different machines.
+> it may cause unexpected issues when editing a file on different machines. 
+> Don't hesitate to read more about this issue on your own time
```

Comment:
"You can read more about this issue" is a cleaner statement than "Don't hesitate to read more about this issue on your own time." Recommend restoring original phrasing, but moving to the new location (line 49 rather than line 68).

Reply:
@tkphd I agree with your review. I'll restore the original phrasing "You can read more about this issue. What about the end where I've added "on your own time" to indicate to the instructor to move on?

Reply:
Might I suggest something like "Though it is beyond the scope of this lesson, you can read more about this issue ...". That way it's clear to new instructors to not get tripped up explaining this in the lesson.

Reply:
And yes, I agree with adding the "on your own time" to the end. It's clarifying and helpful.

Reply:
Oh that's good! Thanks @munkm

Reply:
The new phrasing is much better. Thanks @BTopcuoglu!